### PR TITLE
Fill out README

### LINKED ISSUE #20: Flesh out README
Add quickstart, heroku setup, and configuration for edx-platform to README. Also badges.

### PR DESCRIPTION
**Pull request by carsongee**
closes #20, but it won't be correct until several issues are resolved.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,5 +1,66 @@
...
+authentication to LMod via a proper certificate.  To do that,
+determine the path to your certificate and export that path for the
+configuration with a configuration environment variable: ``export
+LMODP_CERT=/path/to/mycert.pem``.  The certificate needs to be a plain
+(no passphrase) base64 encoded file with both your certificate and
+private key from MIT.
+
+
+Running on Heroku
+=================
```
Comment (othercarson) [inline README.rst]: Don't you need to have heroku support already, like in #16 ?

File: README.rst [hunk 2]
```diff
@@ -1,5 +1,66 @@
...
+
+There is an included Procfile to run the application in heroku, to do
+so just create a heroku app, configure the appropriate environment
+variables in heroku via ``heroku config:set LMODP_...=...``. The one
+catch being that your certificate file and HTPASSWD files probably
+shouldn't be checked in with the repository on heroky, so you can
+actually copy your entire HTPASSWD file into the ``LMODP_HTPASSWD``
+variable and your certificate into ``LMODP_CERT_STRING`` variable.
```
Comment (othercarson) [inline README.rst]: This won't work until #23 lands, duh

**Comment by othercarson** [comment]
Other than the comments above, looks good @carsongee

**Comment by carsongee** [comment]
Thanks for the review @othercarson, I appreciate the feedback

File: README.rst [hunk 3]
```diff
@@ -1,5 +1,66 @@
...
+
+
+Running on Heroku
+=================
+
+There is an included Procfile to run the application in heroku, to do
+so just create a heroku app, configure the appropriate environment
+variables in heroku via ``heroku config:set LMODP_...=...``. The one
+catch being that your certificate file and HTPASSWD files probably
+shouldn't be checked in with the repository on heroky, so you can
```
Comment (bdero) [inline README.rst]: s/heroky/heroku/ ?

**Comment by bdero** [comment]
Looks good to me after the one comment :eyes:

**Pull request merged**